includes/fields/class-date.php

```diff
@@ -267,7 +267,11 @@ protected function normalize() {
 
 		if ( ! is_null( $this->value ) ) {
 			if ( $this->multiple && ! is_array( $this->value ) ) {
-				$this->value = explode( ',', $this->value );
+				if ( strpos( $this->value, ';' ) !== false ) {
+					$this->value = explode( ';', $this->value );
+				} elseif ( strpos( $this->value, ',' ) !== false ) {
+					$this->value = explode( ',', $this->value );
```

Comment:
this fallback may not be required, if I remember correctly the string value is not stored, an array is stored so after the update the dates will be displayed with ; or I'm missing something?

Reply:
If remove this fallback then error appears on save listing. I think it happens because on listing save action we have string data type and normalize function is used in this case too. So as we see in the condition above that if multiple and string data type then we need to convert it into array. But in database it saves as array